src/Sql/Sql/Common/ResourceIdentityHelper.cs

```diff
@@ -21,12 +21,12 @@ public enum ResourceIdentityType
 
     public class ResourceIdentityHelper
     {
-        public static Management.Sql.Models.ResourceIdentity GetIdentityObjectFromType(bool assignIdentityIsPresent)
+        public static Management.Sql.Models.ResourceIdentityWithUserAssignedIdentities GetIdentityObjectFromType(bool assignIdentityIsPresent)
```

Comment:
Why are we changing these?

Reply:
these were changed in the SDK, since we are using the newer version of SDK I had to change them to match